stateDOF: 6 #degree of freedom of state
observationDOF: 4 #degree of freedom of observation
beta: 0.785  #angle of the propellant to the body
g: 3.711 #Gravity of Mars
rho: 0.02 #Planet surface atmosphere density 
C_D: 0.2 #Drag Coefficient
m_b: 1510.0 #mass of EDL system + rover (w/o heatshield or backshell)
m_f: 390.0 #mass of fuel
w_b: 3.2 #width of the sky crane body
h_b: 2.5 #height of the sky crane body
d_b: 2.9 #depth of the sky crane body
w_f: 1.0 #width of the propellant housing
h_f: 0.5 #height of the propellant housing
d_f: 1.0 #housing depth

t_start: 0.0 #start of the simulation time
t_end: 10.0 #end of the simulation time
dt: 0.1

#initial state
xi0: 20.0
xidot0: 0.0
z0: 0.0
zdot0: 0.0
theta0: 0.0
thetadot0: 0.0
P0: identity #just choose "identity" or "zero" as initial state covariance

Nsimruns: 8 #Number of simulation runs. To calculate NEES and NIS
#Number of maximum iterations of EKF equals to (t_end - t_start)/dt

dim_pn: 3 #dimension of process noise
dim_on: 4 #dimension of observation noise
pnoise: [0.0001, 0.0001, 0.0001] #name pn or on has confliction with some build in name, do not use pn or on
onoise: [25.0, 0.25, 0.000025, 0.000225] #first element is set as 25 in matlab, won't be too large?

optimizationChoice: processNoise #optimize "processNoise" or "observationNoise" or "all"
costChoice: JNEES #"JNIS" or "JNEES", which means we compute JNIS or JNEES as cost function
